pkg/connector/resource_types.go

```diff
@@ -0,0 +1,23 @@
+package connector
+
+import (
+	v2 "github.com/conductorone/baton-sdk/pb/c1/connector/v2"
+)
+
+var (
+	resourceTypeUser = &v2.ResourceType{
+		Id:          "user",
+		DisplayName: "User",
+		Traits: []v2.ResourceType_Trait{
+			v2.ResourceType_TRAIT_USER,
+		},
+		Annotations: annotationsForUserResourceType(),
```

Comment:
We should probably remove this indirection and put the annotations here. It's not like anything else is going to call annotationsForUserResourceType().